code/go/internal/validator/test/packages/missing_data_stream/kibana/visualization/good-visualization-abc-1.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
you can probably skip / remove all the kibana files as they should not be needed for the test package.